esmvaltool/utils/cmorizers/obs/cmorize_obs.py

```diff
@@ -213,7 +213,7 @@ def _cmor_reformat(config, obs_list):
     raw_obs = config["rootpath"]["RAWOBS"][0]
 
     # set the reformat scripts dir
-    reformat_scripts = os.path.dirname(__file__)
+    reformat_scripts = os.path.dirname(os.path.abspath(__file__))
     run_dir = os.path.join(config['output_dir'], 'run')
```

Comment:
not really needed since `os.path.dirname(__file__)` will always return the full path in the environment

Reply:
I added it as an additional safety, since on our cluster it was failing in retrieving the full path.